apps/api/src/app/integrations/usecases/check-integration/check-integration.usecase.ts

```diff
@@ -21,7 +21,7 @@ export class CheckIntegration {
         );
       }
 
-      throw e;
+      throw new BadRequestException(e.message);
```

Comment:
Throw a proper validation error, previously it was filtered by the new exception pipe middleware